Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceUpdateExecutor.cpp

```diff
@@ -72,6 +72,20 @@ namespace AzToolsFramework
             AZ::Interface<InstanceUpdateExecutorInterface>::Unregister(this);
         }
 
+        void InstanceUpdateExecutor::AddInstanceToQueue(InstanceOptionalReference instance)
+        {
+            if (!instance.has_value())
+            {
+                AZ_Warning(
+                    "Prefab", false, "InstanceUpdateExecutor::AddInstanceToQueue - "
+                    "Caller tried to insert null instance into the queue.");
+
+                return;
+            }
+
+            m_instancesUpdateQueue.emplace_back(&instance->get());
```

Comment:
nit : use &(instance->get()) to avoid running into any issues with incorrect referencing. I've seen that happen for some AR builds in the past.

Reply:
GP!